_includes/pbc/all/install-features/202212.0/install-the-agent-assist-feature.md

```diff
@@ -0,0 +1,528 @@
+
+
+This document describes how to integrate the [Agent Assist](/docs/pbc/all/user-management/{{site.version}}/agent-assist-feature-overview.html) feature into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the feature core.
+
+### Prerequisites
+
+To start feature integration, overview and install the necessary features:
+
+| NAME         | VERSION          | INTEGRATION GUIDE                                                                                                                                                               |
+|--------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core | {{site.version}} | [Spryker core feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/spryker-core-feature-integration.html)                                            |
+| Product      | {{site.version}} | [Install the Spryker Core Back Office feature](/docs/pbc/all/identity-access-management/{{site.version}}/install-and-upgrade/install-the-spryker-core-back-office-feature.html) |
+| Cart         | {{site.version}} | [Customer Account Management](/docs/scos/dev/feature-integration-guides/{{site.version}}/customer-account-management-feature-integration.html)                                  |
+
+### 1) Install the required modules using composer
+
+Run the following command to install the required modules:
+
+```bash
+composer require spryker-feature/agent-assist:"{{site.version}}" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the following modules have been installed:
+
+| MODULE                          | EXPECTED DIRECTORY                               |
+|---------------------------------|--------------------------------------------------|
+| Agent                           | vendor/spryker/agent                             |
+| AgentGui                        | vendor/spryker/agent-gui                         |
+| SessionAgentValidation          | vendor/spryker-shop/session-agent-validation     |
+| SessionAgentValidationExtension | spryker-shop/session-agent-validation-extension  |
```

Comment:
Extension modules are usually excluded from the verification since their installation is not a question of whether the developer did something wrong, but just composer functioning.